shell/platform/darwin/ios/ios_external_texture_metal.mm

```diff
@@ -65,9 +68,123 @@
     return nullptr;
   }
 
+  sk_sp<SkImage> image = nullptr;
+  if (pixelFormat == kCVPixelFormatType_420YpCbCr8BiPlanarVideoRange ||
+      pixelFormat == kCVPixelFormatType_420YpCbCr8BiPlanarFullRange) {
+    image = WrapYUVAExternalPixelBuffer(pixel_buffer, context);
+  } else {
+    image = WrapRGBAExternalPixelBuffer(pixel_buffer, context);
+  }
+
+  if (!image) {
+    FML_DLOG(ERROR) << "Could not wrap Metal texture as a Skia image.";
+  }
+
+  return image;
+}
+
+sk_sp<SkImage> IOSExternalTextureMetal::WrapYUVAExternalPixelBuffer(
+    fml::CFRef<CVPixelBufferRef> pixel_buffer,
+    GrDirectContext* context) const {
   auto texture_size =
       SkISize::Make(CVPixelBufferGetWidth(pixel_buffer), CVPixelBufferGetHeight(pixel_buffer));
+  CVMetalTextureRef y_metal_texture_raw = NULL;
+  {
+    auto cv_return =
+        CVMetalTextureCacheCreateTextureFromImage(kCFAllocatorDefault,    // allocator
```

Comment:
Please use checked argument name comments (these are checked by the linter):
```c++
CVMetalTextureCacheCreateTextureFromImage(/*allocator=*/ kCFAllocatorDefault,
```

Reply:
This is also to keep the style consistent with the original code. Do I need to modify the original code according to your suggestions?

Reply:
https://google.github.io/styleguide/cppguide.html#Function_Argument_Comments

Reply:
done